utilities/install-collector.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+# netdata
+# real-time performance and health monitoring, done right!
+# (C) 2022
+
+# This script installs community maintained collectors from netdata/community github repo
+
+NETDATA_DIR="/etc/netdata"
+
+if [ "$EUID" -ne 0 ]
+  then echo "Please run as root."
+  exit
```

Comment:
Considering this is an error, it is better to pass a number different of zero as argument to give information about error.